docs/README.md

```diff
@@ -1568,44 +1600,50 @@ authorize them to access protected resources.
 To learn how to include the access token in your requests, check the section [Authorizing operations](#authorizing-operations)
 
 #### Authentication API
+
 The authentication API consists of several endpoints that allow you to manage user registrations, sessions, tokens and more.
 
 The base URL of all these endpoints is the `httpURL` output of your application. See the ["Application Outputs"](#application-outputs) section to know more.
 
 ##### Sign-up
+
 Users can use this endpoint to register in your application and get a role assigned to them.
 
 ![confirmation email](./img/sign-up-verificaiton-email.png)
 ![email confirmed](./img/sign-up-confirmed.png)
 
 ###### Endpoint
+
 ```http request
 POST https://<httpURL>/auth/sign-up
 ```
+
 ###### Request body
+
 ```json
 {
   "clientId": "string",
   "username": "string",
   "password": "string",
   "userAttributes": {
-   	"role": "string"
+    "role": "string"
   }
 }
 ```
 
-Parameter | Description
---------- | -----------
-_clientId_ | The application client Id that you got as an output when the application was deployed.
-_username_ | The username of the user you want to register. It **must be an email**.
-_password_ | The password the user will use to later login into your application and get access tokens.
-_userAttributes_ | Here you can specify the attributes of your user. These are: <br/> -_**role**_:  A unique role this user will have. You can only specify here a role where the `signUpOptions` property is not empty.
-
+| Parameter        | Description                                                                                                                                                                                          |
+| ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _clientId_       | The application client Id that you got as an output when the application was deployed.                                                                                                               |
+| _username_       | The username of the user you want to register. It **must be an email**.                                                                                                                              |
+| _password_       | The password the user will use to later login into your application and get access tokens.                                                                                                           |
+| _userAttributes_ | Here you can specify the attributes of your user. These are: <br/> -_**role**_: A unique role this user will have. You can only specify here a role where the `signUpOptions` property is not empty. |
```

Comment:
What happened with all these changes in format? Does this affect the rendered Markdown?